modules/xdcr-reference/pages/xdcr-advanced-settings.adoc

```diff
@@ -95,6 +95,8 @@ For documents that are smaller, XDCR replicates the document to the target witho
 Note that a low setting risks increased latency, due to a higher number or metadata fetches; but may also reduce the number of required replications (due to source and target having identical copies of the document).
 A high setting lowers latency during replication, since the number of metadata fetches is reduced; but may also raise the replication-rate excessively, overwhelming either network or target cluster.
 
+XDCR optimistic replication is applicable only when the `enableCrossClusterVersioning` property is disabled. When the `enableCrossClusterVersioning` property is set to `true` (enabled), the XDCR processing always retrieves metadata from the target cluster for every document of any size that undergoes replication.
```

Comment:
Part of this description is no longer true -- at least, it depends on the scenario since we've introduced other optimizations using HLV.  So, for simplicity, we should just say (just keep the first sentence and remove the rest):

XDCR optimistic replication is applicable only when the `enableCrossClusterVersioning` property is disabled.